scout/utils/scout_requests.py

```diff
@@ -61,13 +57,34 @@ def fetch_resource(url):
     Returns:
         lines(list(str))
     """
-    try:
-        data = get_request(url)
-        lines = data.split("\n")
-    except Exception as err:
-        raise err
+    data = None
+    if url.startswith("ftp"):
```

Comment:
Which is the case that leads you here specifically? What file is downloaded from an ftp?

Reply:
It is both exac pli and hgnc. requests package does not handle ftp-requests so I figured this was the best solution